File: 07/elixir/main.exs
defmodule Conversion do
  def meters(l, w), do: Float.round(((l * w) * 0.09290304), 3)
end

{len, _} = Integer.parse IO.gets "What is the length of the room in feet?\n"
{width, _} = Integer.parse IO.gets "What is the width of the room in feet?\n"
IO.puts "You entered dimensions of #{len} by #{width} feet."
IO.puts "The area is #{len * width} feet or #{Conversion.meters(len, width)} meters."

File: 07/main.swift
import Foundation

func input() -> String {
	let inputString = NSFileHandle.fileHandleWithStandardInput().availableData
	let strData = NSString(data: inputString, encoding: NSUTF8StringEncoding)!
	return strData.stringByTrimmingCharactersInSet(NSCharacterSet.newlineCharacterSet())
}

print("What is the length of the room in feet?")
let length = Int(input())!
print("What is the width of the room in feet?")
let width = Int(input())!
let area = width * length
let meters = Double(area) * 0.09290304

print("The area is \n \(area) square feet \n \(meters) square meters.")
